packages/sdk-core/scripts/rename-folder.mjs

```
@@ -0,0 +1,174 @@
+
+import { createAxiosInstance } from '../../api-client/dist/index.mjs';
+import {
+  loadVaultKeyBlob,
+  loadFolderMetadata,
+  renameInFolder,
+  updateFolderMetadataAndPublish,
+} from '../dist/index.mjs';
+import {
+  deriveVaultIpnsKeypair,
+  clearBytes,
+} from '../../crypto/dist/index.mjs';
```

ESLint

[error] Replace `⏎··deriveVaultIpnsKeypair,⏎··clearBytes,⏎` with `·deriveVaultIpnsKeypair,·clearBytes·`

(prettier/prettier)